packages/web/src/components/action-drawer/ActionDrawer.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { MouseEventHandler, ReactNode } from 'react'
+import { MouseEventHandler, ReactNode } from 'react'
+import * as React from 'react'
```

Comment:
Would be nice to change the `* as React`s to named imports too!

Reply:
yeah down to do this for sure ^^